manubot/cite/doi.py

```diff
@@ -42,7 +42,7 @@ def get_doi_citeproc(doi):
     citeproc['URL'] = f'https://doi.org/{doi}'
     short_doi_url = get_short_doi_url(doi)
     if short_doi_url:
-        citeproc['short_url'] = short_doi_url
+        citeproc['URL'] = short_doi_url
```

Comment:
If I follow correctly, this will be the only URL that is displayed per the Manubot CSL style.  I prefer the longer URL for DOIs because the DOI can be easily extracted from the reference list.

Reply:
Yes that is true, but I was also hoping to update the default style so a reference renders differently.

Currently:

33. **Sci-Hub provides access to nearly all scholarly literature**
Daniel S Himmelstein, Ariel Rodriguez Romero, Jacob G Levernier, Thomas Anthony Munro, Stephen Reid McLaughlin, Bastian Greshake Tzovaras, Casey S Greene
_eLife_ (2018-03-01) https://doi.org/10.7554/elife.32822

Envisioned:

34. **Sci-Hub provides access to nearly all scholarly literature**
Daniel S Himmelstein, Ariel Rodriguez Romero, Jacob G Levernier, Thomas Anthony Munro, Stephen Reid McLaughlin, Bastian Greshake Tzovaras, Casey S Greene
_eLife_ (2018-03-01) https://doi.org/ckcj
DOI: 10.7554/elife.32822 · PMID: 29424689 · PMCID: PMC5832410

Now possible due to https://github.com/greenelab/manubot/issues/45. This would make a more uniform reference across DOI and PMID / PMCID citations.

A style could always decide to recreate the URL from a DOI if available, if they wanted full DOI links. @agitter what do you think?

Reply:
> A style could always decide to recreate the URL from a DOI if available, if they wanted full DOI links.

Is that possible using the CSL style alone?  If it is, then switching to the short DOI URLs and new default style sounds good to me.  If it would require editing the CSL JSON or some other processing outside the style file, then I'm not so sure.

Reply:
> Is that possible using the CSL style alone?

Yes.

Reply:
I still have a slight preference for the longer URL, but I'm okay switching to the short URL given that it can be adjusted from the style file..